docs/config/index.md

```diff
@@ -139,6 +139,8 @@ export default defineConfig(async ({ command, mode }) => {
 
   - Replacements are performed only when the match is surrounded by word boundaries (`\b`).
 
+  - Since dev and build implement `define` differently, to avoid inconsistencies, we should avoid some use cases, such as defining object attribute names.
+
```

Comment:
Maybe we need an example here? I don't think users will be able to understand if not.

Reply:
done